### PR TITLE
Linking mentions of atomic module to doc

### LINKED ISSUE #2088: Refer to both tutorials and API references as links
1. Outside of code blocks, transform all references to standard manual modules, types and functions into links
2. Sentences referring to tutorials matching an standard library module should also contain a direct link to the API reference

Examples:

In [https://ocaml.org/docs/basic-data-types](../blob/main/data/tutorials/language/0it_01_basic_datatypes.md)

Good, links:

```md
Operations on `float` are provided by the [`Stdlib`](/api/Stdlib.html) and the [`Float`](/api/Float.html) modules.
```

Bad, no links:

```md
The predefined function `fst` returns the first element of a pair, while `snd` returns the second element of a pair.
```

### PR DESCRIPTION
**Pull request by rmeis06**
Hello!

This should partially fix https://github.com/ocaml/ocaml.org/issues/2088

@cuihtlauac @SaySayo Does this look alright?

### CODE REVIEW

File: data/news/multicore/multicore-2020-03.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Onto the details! The various ongoing and completed tasks for Multicore OCaml ar
 * [ocaml-multicore/ocaml-multicore#240](https://github.com/ocaml-multicore/ocaml-multicore/pull/240)
   Proposed implementation of threads in terms of Domain and Atomic
 
-  A new implementation of the `Threads` library for use with the new `Domain` and `Atomic` modules in Multicore OCaml has been proposed. This builds Dune 2.4.0 which in turn makes it useful to build other packages. This PR is open for review.
+  A new implementation of the `Threads` library for use with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml has been proposed. This builds Dune 2.4.0 which in turn makes it useful to build other packages. This PR is open for review.
```
Comment (christinerose) [inline data/news/multicore/multicore-2020-03.md]:
```suggestion
  A new implementation of the `Threads` library (for use with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml) has been proposed. This builds Dune 2.4.0, which in turn makes it useful to build other packages. This PR is open for review.
```
Alternatively:
  A new implementation of the `Threads` library, which will be used with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml, has been proposed. This builds Dune 2.4.0, which in turn makes it useful to build other packages. This PR is open for review.

Comment (cuihtlauac) [reply data/news/multicore/multicore-2020-03.md]: Without brackets is always better

Comment (rmeis06) [reply data/news/multicore/multicore-2020-03.md]: Looks good! Thank you for reviewing :)

**Review by cuihtlauac — APPROVED** [review]
Thanks @rmeis06

**Pull request merged**